demo/examples/Control-Flow/3_project.svg

```diff
@@ -0,0 +1,31 @@
+@startuml
+
+
+control "Foo" as _0
+skinparam maxMessageSize 250
+== Foo <- todos ==
+[->_0 : todos
+activate _0
+opt notfound
+ [<- -_0 : ResourceNotFoundError
+end
+group else if failed
+ [<- -_0 : ErrorResponse
+end
+group else
+ [<- -_0 : Todo
+end
+deactivate _0
+@enduml
+
+PlantUML version 1.2019.13(Tue Dec 10 17:18:29 UTC 2019)
+(GPL source distribution)
+Java Runtime: Java(TM) SE Runtime Environment
+JVM: Java HotSpot(TM) 64-Bit Server VM
+Java Version: 1.7.0_25-b15
+Operating System: Linux
+Default Encoding: UTF-8
+Language: en
+Country: US
+--></g></svg>
```

Comment:
Missing \n

Reply:
Wait this is an auto generated file?

Reply:
Yeah, I've got a feeling this should be a bug report or something